tests/src/CMakeLists.txt

```diff
@@ -34,7 +36,11 @@ foreach(programSourceFile ${programSourceFiles})
   endif()
   dune_add_test(SOURCES ${programSourceFile} LINK_LIBRARIES ikarus LABELS cpp)
```

Comment:
Can you try the same mechanism as below with if(${programName} IN_LIST TEST_NEED_MORE_TIME)
```cmake
  if(${programName} IN_LIST TEST_NEED_MORE_TIME)
dune_add_test(SOURCES ${programSourceFile} LINK_LIBRARIES ikarus LABELS cpp;cpp_slow)
    set_tests_properties(${programName} PROPERTIES TIMEOUT 5000)
  else()
dune_add_test(SOURCES ${programSourceFile} LINK_LIBRARIES ikarus LABELS cpp;cpp_slow)
    set_tests_properties(${programName} PROPERTIES TIMEOUT 1000)
  endif()
```
not sure about the correct syntax though. At the end you should get a new target `build_cpp_slow_tests`

Reply:
In the `else()` part, I have removed `cpp_slow` because `cpp_slow` should only add tests which are in the category `TEST_NEED_MORE_TIME`. Is this right @rath3t ?